README.md

```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+A programming object that represents html elements of a webpage. 
+
 2. What is an event?
 
+Things that happen to the html elements.
+
 3. What is an event listener?
 
+A method on an element that will take two arguments, first one to listen to, and second is the callback for when the event is triggered.
+
 4. Why would we convert a NodeList into an Array?
 
+The NodeList will create a new Array from an existing one.
+
 5. What is a component?
 
+Components are the the building blocks for the web development.
```

Comment:
A component is kinda like the building block for web dev. 

They are also, more accurately, chunks of reusable code. That code can contain definitions, logic, expressions, etc.